storm/String.cpp

```diff
@@ -460,98 +487,76 @@ const char* SStrStr(const char* string, const char* search) {
 }
 
 void SStrTokenize(const char** string, char* buffer, size_t bufferchars, const char* whitespace, int32_t* quoted) {
```

Comment:
Appreciate the much tidier goto-free code! Any objections if we toss the goto-free refactor into a separate PR?